components/brave_wallet_ui/common/hooks/use-multi-chain-sell-assets.ts

```diff
@@ -113,6 +113,11 @@ export const useMultiChainSellAssets = () => {
         asset.chainId === token.chainId)
   }, [options.allAssetOptions])
 
+  const onCloseSellModal = React.useCallback(() => {
+    setSellAmount('')
+    setShowSellModal(false)
+  },[])
```

Comment:
Can we move the modal state stuff outside of this hook?

Reply:
Moved out!